updater/Gemfile.lock

```diff
@@ -322,4 +322,4 @@ DEPENDENCIES
   webmock (~> 3.18)
 
 BUNDLED WITH
-   2.4.11
+   2.3.26
```

Comment:
We don't want this regression, I suspect this is simply a matter of passing a param to `ruby/setup-ruby` action, lemme open a PR.

Reply:
https://github.com/dependabot/dependabot-core/pull/7230

Reply:
This is fixed by https://github.com/dependabot/dependabot-core/pull/7239